### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by SonyaPakhomova**

### CODE REVIEW

**Review by SergiiSV — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+This project is implementation of a taxi service that is connected to a database. The project is built on the principles
+of SOLID and has three levels architecture:
+
+1. Data access layer (DAO) 
+2. Application layer (service)
+3. Presentation layer (controllers).
+
+This project provides the ability to authenticate the user as a driver, add cars and data on their manufacturers. It is
+also possible to add drivers for the correct cars and display the available data from the database.
```
Comment (Nick97-git) [inline README.md]:
```suggestion
This project provides the ability to authenticate the user as a driver, add cars and manufacturers. It is
also possible to add drivers for cars and display all available info about drivers, cars, and manufacturers.
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+## Used technologies: :technologist:
+
+- Maven
+- Maven Checkstyle Plugin
```
Comment (Nick97-git) [inline README.md]: remove this line

File: README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+1. Configure Apache Tomcat (v9.0.50)
+2. Install MySQL and MySQL Workbench
+3. You can use the script from init_db.sql in MySQL Workbench or can create schema by using
+diagram of DB from the picture below by yourself.
+4. In the ConnectionUtil change the constant field data properties to the ones you
+specified when installing MySQL
+5. Starting tomcat, click on "Register" to add a new driver.
+6. In the log4j2.xml at line 7 change fileName path with absolute path to file
+
+![Diagram](https://mate-academy-images.s3.eu-central-1.amazonaws.com/car_diagram_db_2_4f50942103.png)
```
Comment (Nick97-git) [inline README.md]: let's remove this image or change for your own, it's quite popular)

File: README.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+- HTML, CSS
+- Lombok (v1.18.20)
+
+## Instructions for setup :heavy_check_mark:
+
+1. Configure Apache Tomcat (v9.0.50)
+2. Install MySQL and MySQL Workbench
+3. You can use the script from init_db.sql in MySQL Workbench or can create schema by using
+diagram of DB from the picture below by yourself.
+4. In the ConnectionUtil change the constant field data properties to the ones you
```
Comment (Nick97-git) [inline README.md]:
```suggestion
4. In the ConnectionUtil change data properties to the ones you
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+specified when installing MySQL
+5. Starting tomcat, click on "Register" to add a new driver.
```
Comment (Nick97-git) [inline README.md]: remove this paragraph

File: README.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+6. In the log4j2.xml at line 7 change fileName path with absolute path to file
```
Comment (Nick97-git) [inline README.md]:
```suggestion
6. In the log4j2.xml change fileName path with the absolute path to file
```

**Review by Nick97-git — CHANGES_REQUESTED** [review]
- DeleteCarController, DeleteDriverController, DeleteManufacturerController -> remove /all from /cars/all, /drivers/all, /manufacturers/all url patterns
- remove redundant / from urls like /cars/, /drivers/, /manufacturers/
- move GetMyCurrentCarsController to `cars` package
- DriverDaoImpl -> move private method after public ones

File: README.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+## Instructions for setup :heavy_check_mark:
+
+1. Configure Apache Tomcat (v9.0.50)
+2. Install MySQL and MySQL Workbench
+3. You can use the script from init_db.sql in MySQL Workbench or can create schema by using
+diagram of DB from the picture below by yourself.
+4. In the ConnectionUtil change data properties to the ones you
+specified when installing MySQL
+6. In the log4j2.xml change fileName path with the absolute path to file
```
Comment (Nick97-git) [inline README.md]:
```suggestion
5. In the log4j2.xml change fileName path with the absolute path to file
```

File: README.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+- JSTL
+- HTML, CSS
+- Lombok (v1.18.20)
+
+## Instructions for setup :heavy_check_mark:
+
+1. Configure Apache Tomcat (v9.0.50)
+2. Install MySQL and MySQL Workbench
+3. You can use the script from init_db.sql in MySQL Workbench or can create schema by using
+diagram of DB from the picture below by yourself.
```
Comment (Nick97-git) [inline README.md]: you removed digram so modify this sentence too

File: README.md (hunk 2 above)
Comment (Nick97-git) [reply README.md]: you missed this one

**Review by Nick97-git — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Nick97-git) [reply README.md]: you still didn't remove this line)

File: README.md [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+- Servlet
+- JSP
+- JSTL
+- HTML, CSS
+- Lombok (v1.18.20)
+
+## Instructions for setup :heavy_check_mark:
+1. Configure Apache Tomcat (v9.0.50)
+2. Install MySQL and MySQL Workbench
+3. You can use the script from init_db.sql in MySQL Workbench..
```
Comment (Nick97-git) [inline README.md]: for what do they need to use this script?

**Review by Nick97-git — CHANGES_REQUESTED** [review]

**Comment by Nick97-git** [comment]
what about the last comment?

**Review by Nick97-git — APPROVED** [review]

**Pull request merged**